docs/source/user_guide/automl.ipynb

```diff
@@ -642,6 +642,150 @@
+    "\n",
+    "3. Runn all pipelines with preprocessing components:\n",
+    "    a. scan rest of estimators (`IterativeAlgorithm` batch 1).\n",
+    "4. First ensembling run\n",
+    "\n",
+    "Fast mode ends here. Begin long mode.\n",
+    "\n",
+    "6. Run top 3 estimators:\n",
+    "    a. Generate 50 random parameter sets. Run all 150 in one batch\n",
+    "    \n",
+    "7. Second ensembling run\n",
+    "\n",
+    "8. Repeat these indefinitely until stopping criterion is met:\n",
+    "    a. For each of the previous top 3 estimators, sample 10 parameters from the tuner. Run all 30 in one batch\n",
+    "    b. Run ensembling\n",
+    "    \n",
+    "To this end, it is recommended to use the top level `search()` method to run `DefaultAlgorithm`. This allows users to specify running search with just the `mode` parameter, where `fast` is recommended for users who want a fast scan at how EvalML pipelines will perform on their problem and where `long` is reserved for a deeper dive into high performing pipelines. One can also specify `automl_algorithm='default'` using `AutoMLSearch` and it will default to using `fast` mode. Users are welcome to select `max_batches` according to the algorithm above (or other stopping criteria) but should be aware that results may not be optimal if the algorithm does not run for the full length of `fast` mode."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.automl import search\n",
+    "\n",
+    "# top level search method will run `AutoMLSearch` with `IterativeAlgorithm` as well as apply our default data checks\n",
+    "automl_default, messages_default = search(X, y, problem_type='binary', mode='fast')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml import AutoMLSearch\n",
+    "aml_default = AutoMLSearch(X_train=X, y_train=y, problem_type='binary', automl_algorithm='default', verbose=True)\n",
+    "aml_default.search()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pipeline differences\n",
+    "Through the search output above, we can see how pipelines differ between `IterativeAlgorithm` and `DefaultAlgorithm`. This is because `DefaultAlgorithm` utilizes new components such as `RFRegressorSelectFromModel` and other column selectors for feature selection as well as a new pipeline structure to handle feature selection for categorical and non-categorical features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aml_iterative.best_pipeline.graph()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aml_default.best_pipeline.graph()"
```

Comment:
A note for the future: this graph is _impossible_ to read since it's so large. We should find some way in both our docs and our code that lets us zoom in to see better!